File: TOMIMSV4/Entity/Interface/Transacciones_Push/clsBeI_nav_transacciones_push.vb
Public Class clsBeI_nav_transacciones_push
    Implements ICloneable

    Public Property IdTransaccionPush() As Integer = 0
    Public Property IdEmpresa() As Integer = 0
    Public Property IdBodega() As Integer = 0
    Public Property IdPropietariobodega() As Integer = 0
    Public Property IdOrdenCompra() As Integer = 0
    Public Property IdRecepcionEnc() As Integer = 0
    Public Property IdRecepcionDet() As Integer = 0
    Public Property Idproductobodega() As Integer = 0
    Public Property Idproducto() As Integer = 0
    Public Property Idunidadmedida() As Integer = 0
    Public Property Idpresentacion() As Integer = 0
    Public Property Idproductoestado() As Integer = 0
    Public Property Cantidad() As Double = 0.0
    Public Property Peso() As Double = 0.0
    Public Property Lote() As String = ""
    Public Property Fecha_vence() As Date = New Date(1900, 1, 1)
    Public Property No_linea() As String = ""
    Public Property Codigo_variante() As String = ""
    Public Property Nom_unidad_medida() As String = ""
    Public Property Tipo_transaccion() As String = ""
    Public Property IdTipoDocumento() As Integer = 0
    Public Property Tipo_push() As String = ""
    Public Property No_recepcion_almacen() As String = ""
    Public Property Documento_ubicacion() As String = ""
    Public Property Documento_ingreso() As String = ""
    Public Property Documento_recepcion() As String = ""
    Public Property Location_code() As String = ""
    Public Property Zone_code() As String = ""
    Public Property Bin_code() As String = ""
    Public Property Assigne_user_id() As String = ""
    Public Property Item_no() As String = ""
    Public Property No_orden_prod() As String = ""
    Public Property Respuesta_push() As String = ""
    Public Property Enviado_A_ERP() As Boolean = False
    Public Property Fec_agr() As Date = Date.Now
    Public Property User_agr() As String = ""
    Public Property Fec_mod() As Date = Date.Now
    Public Property User_mod() As String = ""

    Sub New()
        '#EJC: Add your constructor here... 
    End Sub

    Public Function Clone() As Object Implements System.ICloneable.Clone
        Return MyBase.MemberwiseClone()
    End Function

End Class

File: WMS.EntityCore/Transacciones/clsBeI_nav_transacciones_push.cs
namespace WMS.EntityCore.Transacciones
{
    public class clsBeI_nav_transacciones_push : ICloneable
    {
        public int IdTransaccionPush { get; set; } = 0;
        public int IdEmpresa { get; set; } = 0;
        public int IdBodega { get; set; } = 0;
        public int IdPropietariobodega { get; set; } = 0;
        public int IdOrdenCompra { get; set; } = 0;
        public int IdRecepcionEnc { get; set; } = 0;
        public int IdRecepcionDet { get; set; } = 0;
        public int Idproductobodega { get; set; } = 0;
        public int Idproducto { get; set; } = 0;
        public int Idunidadmedida { get; set; } = 0;
        public int Idpresentacion { get; set; } = 0;
        public int Idproductoestado { get; set; } = 0;
        public double Cantidad { get; set; } = 0.0;
        public double Peso { get; set; } = 0.0;
        public string Lote { get; set; } = "";
        public DateTime Fecha_vence { get; set; } = new DateTime(1900, 1, 1);
        public string No_linea { get; set; } = "";
        public string Codigo_variante { get; set; } = "";
        public string Nom_unidad_medida { get; set; } = "";
        public string Tipo_transaccion { get; set; } = "";
        public int IdTipoDocumento { get; set; } = 0;
        public string Tipo_push { get; set; } = "";
        public string No_recepcion_almacen { get; set; } = "";
        public string Documento_ubicacion { get; set; } = "";
        public string Documento_ingreso { get; set; } = "";
        public string Documento_recepcion { get; set; } = "";
        public string Location_code { get; set; } = "";
        public string Zone_code { get; set; } = "";
        public string Bin_code { get; set; } = "";
        public string Assigne_user_id { get; set; } = "";
        public string Item_no { get; set; } = "";
        public string No_orden_prod { get; set; } = "";
        public string Respuesta_push { get; set; } = "";
        public bool Enviado_A_ERP { get; set; } = false;
        public DateTime Fec_agr { get; set; } = DateTime.Now;
        public string User_agr { get; set; } = "";
        public DateTime Fec_mod { get; set; } = DateTime.Now;
        public string User_mod { get; set; } = "";

        public clsBeI_nav_transacciones_push()
        {
            // Constructor
        }

        public object Clone()
        {
            return this.MemberwiseClone();
        }
    }
}